packages/react/src/components/UIShell/SwitcherItem.js

```diff
@@ -37,9 +43,28 @@ const SwitcherItem = React.forwardRef(function SwitcherItem(props, ref) {
     [`${prefix}--switcher__item-link--selected`]: isSelected,
   });
 
+  function setTabFocus(evt) {
+    if (match(evt, keys.ArrowDown)) {
+      handleSwitcherItemFocus?.({
+        currentIndex: index,
+        direction: 1,
+      });
+    }
+    if (match(evt, keys.ArrowUp)) {
+      handleSwitcherItemFocus?.({
+        currentIndex: index,
+        direction: -1,
+      });
+    }
```

Comment:
This navigation is working correctly for me. I'd confirm with @mbgower wether we'd want to also keep the tab navigation, I feel like we usually either have the arrow navigation or the tab navigation, not both